doc/releases/release-notes-4.3.rst

```diff
@@ -83,6 +83,10 @@ New Drivers
   Same as above for boards, this will also be recomputed at the time of the release.
   Just link the driver, further details go in the binding description
 
+* Interrupt controller
+
+   * :dtcompatible:`st,stm32-exti`
```

Comment:
I think the paragraph you provided remains valuable in addition to explain users what happens:
```
   The EXTI driver for STM32 has become a separate driver and is now split away
   from the GPIO module. This change paves the way for additional hardware drivers with
   extended interrupt capabilities like rising or falling edge interrupt events.
```
I know this section uses to contain only compatibles, but nothing prevents to provide a little bit of context.

Reply:
Do you mean - just to add the description right after line 87?